apps/odk_publish/models.py

```diff
@@ -98,6 +98,73 @@ class Meta:
 class Project(AbstractBaseModel):
     """A project in ODK Central."""
 
+    APP_LANGUAGE_CHOICES = list(
```

Comment:
Let's include a link to the source for this list, for example:
```suggestion
    # APP_LANGUAGE_CHOICES should be updated only based on the supported
    # values for the "app_language " setting: https://docs.getodk.org/collect-import-export/
    APP_LANGUAGE_CHOICES = list(
```